docs/faq.rst

```diff
@@ -230,6 +230,16 @@ all the part files and manually concatenate them together.
 
 For more details, see :ref:`checkpoints_parts`.
 
+How can I compare contents of the archive to my local filesystem?
+-----------------------------------------------------------------
+
+You can instruct ``export-tar`` to send tar archive to the stdout, and
+then use ``tar`` to perform the comparison:
+
+::
```

Comment:
can you add an empty line below the `::` - iirc it will not format correctly otherwise, see line 114.